### PR TITLE
Enforce naming convention for qe-sap-deployment settings

### PR DESCRIPTION
**Pull request by mpagot**
Rename variables in qesap and Trento. List of renamed settings:

QESAPDEPLOY_VER --> QESAP_INSTALL_VERSION
QESAPDEPLOY_GIT_NO_VERIFY --> QESAP_INSTALL_GITHUB_NO_VERIFY
QESAPDEPLOY_GITHUB_BRANCH --> QESAP_INSTALL_GITHUB_BRANCH
QESAPDEPLOY_GITHUB_REPO --> QESAP_INSTALL_GITHUB_REPO
TRENTO_CLUSTER_OS_VER --> TRENTO_QESAPDEPLOY_CLUSTER_OS_VER
QESAPDEPLOY_SAPCAR --> TRENTO_QESAPDEPLOY_SAPCAR
QESAPDEPLOY_IMDB_SERVER --> TRENTO_QESAPDEPLOY_IMDB_SERVER
QESAPDEPLOY_IMDB_CLIENT --> TRENTO_QESAPDEPLOY_IMDB_CLIENT

Add QESAP variables documentation.
Add qesapdeploy API to get agents number and terraform dir.
Use -f for curl of config and releases
Disable bastion in GCP regression test.

- Verification run:
  -- Tested in Trento http://openqaworker15.qa.suse.cz/tests/59796#
  -- Tested with qesap regression:  AZURE:https://openqaworker15.qa.suse.cz/tests/59811#  https://openqaworker15.qa.suse.cz/tests/59812#    GCP (fails): https://openqaworker15.qa.suse.cz/tests/59810 and https://openqaworker15.qa.suse.cz/tests/59711

### CODE REVIEW

**Review by alvarocarvajald — APPROVED** [review]
LGTM

File: lib/qesapdeployment.pm [hunk 1]
```diff
@@ -244,6 +245,36 @@ sub qesap_get_inventory {
     return "$paths{deployment_dir}/terraform/" . lc $provider . '/inventory.yaml';
 }
 
+=head3 qesap_get_agents_number
+
+Get the number of SAP cluster nodes where the trento-agent has been installed
+=cut
+
+sub qesap_get_agents_number {
```
Comment (lpalovsky) [inline lib/qesapdeployment.pm:253]: Wouldn't it be better to have this function outside of the deployment library since it is trento related?

Comment (mpagot) [reply lib/qesapdeployment.pm:253]: I can change the description to 

```
Get the number of cluster nodes
```

and even the function name to **qesap_get_nodes_number**

The code is not Trento related. The implementation is explained in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15797/files#diff-c4f6960b011dade32b03b0797381a30cb073505ec0cab10ca6c82fdcaba36a6bR177

It is just an inventory.yaml parser that return the number of nodes

Comment (lpalovsky) [reply lib/qesapdeployment.pm:253]: Yeah renaming it to "qesap_get_nodes_number" sounds good.

**Review by lpalovsky — APPROVED** [review]
Aside from function naming comment, LGTM.

**Review by dzedro — APPROVED** [review]
This variables change more frequent than socks.
I can't distinguish if variable is Trento specific or not, will see.

**Review by jankohoutek — APPROVED** [review]
LGTM

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Comment by lilyeyes** [comment]
I only checked the renamed settings, there lgtm.

**Pull request merged**